python/pfs/drp/stella/reduceArcTask.py

```diff
@@ -28,7 +29,20 @@ class ReduceArcConfig(pexConfig.Config):
                                                        doc="Calibrate a SpectrumSet's wavelengths")
     minArcLineIntensity = pexConfig.Field(doc="Minimum 'NIST' intensity to use emission lines",
                                           dtype=float, default=0)
-    doUpdateCenters = pexConfig.Field(dtype=bool, default=True, doc="Update centers from the fiberTrace?")
+    tweakCenters = pexConfig.ChoiceField(
+        dtype=str,
+        doc="Whether to tweak the detectorMap xCenters, and the source for the tweaks",
+        default="ARC",
+        allowed={"NONE": "No tweaking",
+                 "ARC": "Tweak based on the arc lines",
+                 "FIBERTRACE": "Tweak based on the fiberTrace",
+                 },
+    )
+    arcTweakWidth = pexConfig.Field(dtype=int, default=5, doc="Half-width for centroiding in x")
+    arcTweakHeight = pexConfig.Field(dtype=int, default=2, doc="Half-height for stacking when centroiding")
+    arcTweakMask = pexConfig.ListField(dtype=str, default=["BAD", "SAT", "NO_DATA"],
+                                       doc="Mask planes to reject when centroiding arc lines")
+    arcTweakRms = pexConfig.Field(dtype=float, default=2.0, doc="Guess line RMS when centroiding arc lines")
```

Comment:
Very minor comment: don't like the use of the word 'tweak' (or 'hack' in formal class/method/fn/parameter names). How about 'adjust'? Anyway, I won't object to that word being used if there's no better alternative or too much work to rename.